src/configureStore.js

```diff
@@ -1,11 +1,19 @@
 import { createStore, applyMiddleware } from 'redux';
+import createSagaMiddleware from 'redux-saga';
 import createLogger from 'redux-logger';
 
 import rootReducer from './reducers/';
+import sagas from './sagas/index';
```

Comment:
Hm.. I think this explains the redux-saga concept better than I do: 
"The mental model is that a saga is like a separate thread in your application that's solely responsible for side effects. redux-saga is a redux middleware, which means this thread can be started, paused and cancelled from the main application with normal redux actions, it has access to the full redux application state and it can dispatch redux actions as well." 

I think axios is more comparable to superagent (https://github.com/Pubcrawler/weiss/blob/1c775b72c01465854c6a8e8adde12e613ae8484f/src/services/index.js). We could easily use axios :)